### PR TITLE
11.1 log treatments

### PR DESCRIPTION
**Pull request by raeffu**
@sebastianhaeni review!

### CODE REVIEW

**Comment by sebastianhaeni** [comment]
Build fails... Please fix before this get's merged.

**Comment by raeffu** [comment]
Its a travis problem, local build is ok.

File: src/test/java/ch/bfh/healvis/data/model/TreatmentsTest.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+                predicates.add(criteriaBuilder.like(from.get(Appointment_.id), id));
+            }
+
+        });
+
+        appointment = appointments.getItem(appointments.firstItemId()).getEntity();
+    }
+    
+    @Test
```
Comment (sebastianhaeni) [inline src/test/java/ch/bfh/healvis/data/model/TreatmentsTest.java]: Annotations are supposed to be written after JavaDoc as they are part of the code that's been documented.

**Comment by sebastianhaeni** [comment]
Sorry for closing it accidentally. Build now passes.
Why the heck is doc/cs1-task14/readme.md acting weird on this branch?

**Comment by raeffu** [comment]
Cool, thx!

Don't know, there was something weird going on when I rebased this branch with master..

**Pull request merged**